### PR TITLE
Add JS and Wasm targets

### LINKED ISSUE #1: JS/Wasm Support
Hi there!
Would love to see this support JS/Wasm targets.

As far as I'm aware, just like material icons, they're not using any platform-specific APIs (just paths).
So are there some other hurdles blocking this? I can submit a PR.

And thanks for the library!

### PR DESCRIPTION
**Pull request by skaldebane**
Adds JS and Wasm targets, samples, and README updates.
Closes #1.

### CODE REVIEW

**Comment by skaldebane** [comment]
Sample builds took too long and my machine ran out of memory, so I couldn't test them unfortunately.
If everything's alright, they should build and run without issues on a better machine.

**Comment by adamglin0** [comment]
> Sample builds took too long and my machine ran out of memory, so I couldn't test them unfortunately.
> If everything's alright, they should build and run without issues on a better machine.

I will test it.

File: sample/shared/build.gradle.kts [hunk 1]
```diff
@@ -28,6 +29,19 @@ kotlin {
...
+    js(IR) {
+        moduleName = "sharedApp"
+        browser()
+        binaries.executable()
+    }
+
+    @OptIn(ExperimentalWasmDsl::class)
+    wasmJs {
+        moduleName = "sharedApp"
+        browser()
```
Comment (adamglin0) [inline sample/shared/build.gradle.kts]: need output setting. 
e.g.
```
browser {
            commonWebpackConfig {
                outputFileName = "sharedApp.js"
            }
        }
```

File: sample/shared/build.gradle.kts [hunk 2]
```diff
@@ -28,6 +29,19 @@ kotlin {
         }
     }
 
+    js(IR) {
+        moduleName = "sharedApp"
+        browser()
```
Comment (adamglin0) [inline sample/shared/build.gradle.kts]:
```
browser {
            commonWebpackConfig {
                outputFileName = "sharedApp.js"
            }
        }
```

**Review by adamglin0 — CHANGES_REQUESTED** [review]
WASM and JS works good (after add output setting)
image

File: sample/shared/build.gradle.kts (hunk 1 above)
Comment (skaldebane) [reply sample/shared/build.gradle.kts]: ah yeah, i initially thought these extra settings were only for source maps.

**Review by adamglin0 — APPROVED** [review]
LGTM!

**Pull request merged**